nav2_behavior_tree/test/plugins/decorator/distance_controller_test.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2018 Intel Corporation
+// Copyright (c) 2020 Sarthak Mittal
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <cmath>
+#include <memory>
+#include <set>
+
+#include "rclcpp/rclcpp.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include "nav2_util/robot_utils.hpp"
+
+#include "../../transform_handler.hpp"
```

Comment:
I will have to add ```plugins/``` to included directories for this. I was planning on doing this but I think it will require some refactoring. The ```plugins/``` directory will have to be moved to a better location, maybe ```include/nav2_behavior_tree/plugins```.

Reply:
Ah, I see, this can remain as is then. I just wanted to make sure there was a good reason.